### ISSUE TITLE
would you please  suggest code for multi camera multi object tracking evaluation

### ISSUE THREAD

**Issue by ssbilakeri** [cheind/py-motmetrics#157]

**Comment by Hasankanso** [comment]
check mcta metric in http://www.mct2014.com/Evaluation.html

it is implemented in matlab...I wish I can find it in python!

**Comment by whizmo** [comment]
You can also use this library for this purpose by changing the frame index the following way:
frame_ind = frame_ind*1000 + cam_number
This way you can merge the results from all the cameras and use this library.

**Comment by SajjadPSavoji** [comment]
> You can also use this library for this purpose by changing the frame index the following way:
> frame_ind = frame_ind*1000 + cam_number
> This way you can merge the results from all the cameras and use this library.

@angelcarro 
Can you be more detailed? I'm assuming that you are trying to merge results(from multiple camera) by assigning new frame numbers to each source in a way that videos from different sources do not overlap in time.

What does 1000 represents in your solution?

**Comment by angelcarro** [comment]
Yes, the idea is to generate a new frame index to merge the information from the multiple cameras without collisions. This operation is performed both to the ground truth annotations and to the tracking results. 

1000 is just a number bigger than the number of cameras. Using 1000 (or 100) is useful to visually separate the frame index and the camera number.
